core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -78,6 +78,35 @@ object MultiCloudWorkspaceService {
 
   def getStorageContainerName(workspaceId: UUID): String = s"sc-${workspaceId}"
 
+  def buildPolicyInputs(workspaceRequest: WorkspaceRequest): Option[WsmPolicyInputs] = {
```

Comment:
Moved to the companion object to make it clear it was a pure function, and make it easier to test directly.